runtime/kusama/src/weights/pallet_bags_list.rs

```diff
@@ -1,65 +1,64 @@
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
 
-//! Autogenerated weights for pallet_bags_list
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+//! Autogenerated weights for `pallet_bags_list`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2021-09-02, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+//! DATE: 2021-09-17, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("kusama-dev"), DB CACHE: 128
 
 // Executed Command:
-// target/release/substrate
+// target/release/polkadot
 // benchmark
-// --chain=dev
+// --chain=kusama-dev
 // --steps=50
 // --repeat=20
 // --pallet=pallet_bags_list
 // --extrinsic=*
 // --execution=wasm
 // --wasm-execution=compiled
 // --heap-pages=4096
-// --output=./frame/bags-list/src/weights.rs
-// --template=./.maintain/frame-weight-template.hbs
+// --header=./file_header.txt
+// --output=./runtime/kusama/src/weights/
 
-// NOTE: this is copy pasta from substrate % some tweaks
 
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use frame_support::{traits::Get, weights::Weight};
 use sp_std::marker::PhantomData;
 
-/// Weight functions needed for pallet_bags_list.
-
-/// Weights for pallet_bags_list using the Substrate node and recommended hardware.
+/// Weight functions for `pallet_bags_list`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_bags_list::WeightInfo for WeightInfo<T> {
+	// Storage: Staking Bonded (r:1 w:0)
+	// Storage: Staking Ledger (r:1 w:0)
+	// Storage: BagsList ListNodes (r:4 w:4)
+	// Storage: BagsList ListBags (r:1 w:1)
 	fn rebag_non_terminal() -> Weight {
-		(75_718_000 as Weight)
+		(65_491_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(5 as Weight))
 	}
 	// Storage: Staking Bonded (r:1 w:0)
 	// Storage: Staking Ledger (r:1 w:0)
-	// Storage: BagsList ListNodes (r:4 w:4)
-	// Storage: BagsList ListBags (r:1 w:1)
+	// Storage: BagsList ListNodes (r:3 w:3)
```

Comment:
hmm why did the storage changed compared to substrate weights?

Reply:
Its a different benchmark, the terminal nodes r/w bags 2 but nodes 1 less time. The diff is because the old stuff was just copy pasta to get things to compile